lens/client.go

```diff
@@ -0,0 +1,51 @@
+package lens
+
+import (
+	"fmt"
+
+	"github.com/RTradeLtd/config"
+	"github.com/RTradeLtd/grpc/dialer"
+	pb "github.com/RTradeLtd/grpc/lens"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+const (
+	defaultURL = "127.0.0.1:9998"
+)
+
+// Client is a lens client used to make requests to the Lens gRPC server
+type Client struct {
+	pb.IndexerAPIClient
+}
+
+// NewClient is used to generate our lens client
+func NewClient(opts config.Endpoints) (*Client, error) {
```

Comment:
i noticed that the signer client is in the `grpc` folder here, it might be nice to have all the grpc clients used by temporal inside `grpc`? or maybe rename it to `grpc_clients` to avoid conflicts with the grpc repo